Description: fix GCC 4.7 build on 'other' platforms like GNU/kFreeBSD
 Assume that all platforms in the 'other' category will have unistd.h;
 as of GCC 4.7 they are going to have to include it.
Author: Steven Chamberlain <steven@pyro.eu.org>

Index: protobuf/gtest/include/gtest/internal/gtest-port.h
===================================================================
--- protobuf.orig/gtest/include/gtest/internal/gtest-port.h	2013-08-13 17:03:10.932794167 -0400
+++ protobuf/gtest/include/gtest/internal/gtest-port.h	2013-08-13 17:03:10.932794167 -0400
@@ -325,6 +325,9 @@
 
 #else
 
+// Assume other platforms have unistd.h
+#include <unistd.h>  // NOLINT
+
 // <regex.h> may not be available on this platform.  Use our own
 // simple regex implementation instead.
 # define GTEST_USES_SIMPLE_RE 1
